kie-server-parent/kie-server-controller-plugin/README.md

```diff
@@ -28,7 +28,7 @@ On pom.xml of local project add plugin:
     <plugin>
         <groupId>org.kie.server</groupId>
         <artifactId>kie-server-controller-plugin</artifactId>
-        <version>7.5.0-SNAPSHOT</version>
+        <version>7.15.0-SNAPSHOT</version>
```

Comment:
should be 7.16 now or maybe even replace it with ${version.org.kie} just to indicate it should be same version as all other org.kie artefacts

Reply:
changed to ${version.org.kie}